app/models/order.rb

```diff
@@ -0,0 +1,10 @@
+class Order < ApplicationRecord
+  validates_presence_of :user_id, :status
+
+  belongs_to :user
+  belongs_to :order_items
```

Comment:
@Vjp888 @joequincy @tnodland 

Order should not belongs_to OrderItems I believe?